_data/navbars/security.yml

```diff
@@ -79,7 +81,7 @@ section:
   section:
   - title: Configure encryption and authentication
     path: /security/comms/crypto-auth
-  - title: Schedule Typha for scaling to well-known nodes
+  - title: Schedule to well-known nodes
```

Comment:
I'm not sure if you were intending to make this change, or if it was a merge error of some sort.  "Schedule Typha to well-known-nodes" would be ok, but needs to say Typha.

Reply:
yes this was a merge conflict error sorry!